esrally/track/loader.py

```diff
@@ -892,6 +893,43 @@ def on_after_load_track(self, track):
         return track
 
 
+class ServerlessFilterTrackProcessor(TrackProcessor):
+    def __init__(self, cfg):
+        self.logger = logging.getLogger(__name__)
+        self.serverless_mode = cfg.opts("driver", "serverless.mode")
+        self.serverless_operator = cfg.opts("driver", "serverless.operator")
+
+    def _is_filtered_task(self, operation):
+        try:
+            op = track.OperationType.from_hyphenated_string(operation.type)
+            if self.serverless_operator:
+                return op.serverless_status < track.ServerlessStatus.Internal
+            else:
+                return op.serverless_status < track.ServerlessStatus.Public
+        except KeyError:
+            self.logger.info("Treating user-provided operation type [%s] for operation [%s] as public.", operation.type, operation.name)
+            return True
```

Comment:
This should be `False`, otherwise filtering is applied. Can we add a unit test to cover this scenario?

Reply:
Oh wow, thanks. This is likely causing the rally-tracks-compat failures as elastic/logs and elastic/security are failing and are using user-defined operations. Fixed in [`a6c54dc` (#1760)](https://github.com/elastic/rally/pull/1760/commits/a6c54dc4c62aa909a01020b0dd1b10abf222431a)